src/custom/components/ExplorerLink/index.tsx

```diff
@@ -22,5 +21,5 @@ export function ExplorerLink(props: Props) {
   }
 
   const linkLabel = label || getExplorerLabel(chainId, id, type)
-  return <ExternalLink href={getExplorerOrderLink(chainId, id)}>{linkLabel}</ExternalLink>
+  return <ExternalLink href={getEtherscanLink(chainId, id, type)}>{linkLabel}</ExternalLink>
```

Comment:
Well, this fixed the issue but the code seems to be wrong, if we think about the function name.
What about renaming [`getEtherscanLink`](https://github.com/gnosis/gp-swap-ui/blob/release%2F0.6.0/src/custom/utils/index.ts#L83) to `getExplorerLink` ?